README.md

```diff
@@ -100,8 +100,8 @@ Member, CSS WG; Google
 
 -----
 
-**Related reading**
-* [Extend the Web Forward](http://yehudakatz.com/2013/05/21/extend-the-web-forward/), by Yehuda Katz, which fleshes out these ideas more.
-* [An Extensible Approach to Browser Security Policy](http://yehudakatz.com/2013/05/24/an-extensible-approach-to-browser-security-policy/), by Yehuda Katz, which presents a practical application of these principles to the problem of designing an API for the browser's Content Security Policy.
+**Related Reading**
+* [Extend the Web Forward](http://yehudakatz.com/2013/05/21/extend-the-web-forward/), by Yehuda Katz, which fleshes out more these ideas more.
+* [An Extensible Approach to Browser Security Policy](http://yehudakatz.com/2013/05/24/an-extensible-approach-to-browser-security-policy/), by Yehuda Katz, which presents a practical application of these principles to the problem of designing an API for the browser's [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/Security/CSP).
```

Comment:
again, using an English URL because of a [bug](https://bug635044.bugzilla.mozilla.org/show_bug.cgi?id=963943) with the site.